### ISSUE TITLE
default light theme - UI text hard to read

### ISSUE THREAD

**Issue by alfalabs** [Microsoft/vscode#6065]
- VSCode Version: 1.0
- OS Version: Win7

Steps to Reproduce:
select File>Preferences>Color Theme: Light (Visual Studio) or Light + (default light)

Look at the colors in the file explorer in the left panel - not the editor panel.
The text is light grey on slightly lighter grey background.
This is very hard to read.
- how to change that?
- where is the theme file located?
- what to change in this file to set text color to BLACK?

Please help people with weak eyesight.

**Comment by bgashler1** [comment]
@alfalabs thanks for pointing this out.  Are you having trouble seeing just the path on the right, or are you having trouble with both the path and the filename?
<img width="297" alt="screen shot 2016-05-04 at 8 52 06 am" src="https://cloud.githubusercontent.com/assets/11839736/15020504/25ce7a16-11d7-11e6-9492-5a408c6f40fe.png">

While we work to improve the color contrast here, I'd recommend switching to the High Contrast theme.

**Note to self:***
Here are the color contrast measurements.  The right one (B) is to low according to WCAG AA (less than 4.5:1.
![6065-light-theme-explorer-text-hard-to-read](https://cloud.githubusercontent.com/assets/11839736/15020679/fab0ef84-11d7-11e6-934a-dde6d6f025b3.png)

**Comment by bgashler1** [comment]
@alfalabs:
to further answer your questions, in case you really want to modify your light theme (instead of using the high-contrast theme), you can modify it by going to `file:///C:/Program%20Files%20(x86)/Microsoft%20VS%20Code%20Alpha/resources/app/out/vs/workbench/workbench.main.css`

Change the following rule so that the **`opacity` is `1`**.
```
.monaco-file-label .file-path {
    opacity: 1;
    margin-left: .5em;
    font-size: .9em;
}
```

**Comment by alfalabs** [comment]
@bgashler1 
re 1  - having trouble with both, file and path as in your illustration. If text was black `#000` it would be much better. 
re 2 - Yes I need light theme. Changed opacity to 1 as per your instruction (needed admin permission) but I see no difference. Is there a cache for .css file I need to clear? (cleared IE cache but it did not help)

**Comment by bgashler1** [comment]
Sorry @alfalabs, I just updated the path to the CSS in my previous comment.  The one I gave you was for the Alpha build of VS Code (and that's why you saw no change).  It should work now if you work off the correct path I just sent you.

**Comment by alfalabs** [comment]
I did figure it by myself since I do not have `alfa` folder. Still no change after changing opacity. I rebooted my computer just to make sure.

![vscode](https://cloud.githubusercontent.com/assets/11356118/15022811/67f4e8fc-11fb-11e6-9c29-7a645ace02dc.png)

**Comment by bgashler1** [comment]
When you open Toggle Developer Tools from the help menu and inspect the working files text, do you see that its opacity has changed to `opacity:1`?

It works on my machine when I change it.

**Comment by alfalabs** [comment]
@bgashler1 
This is cool! 
I did not know about Developer Tools.
I could not find the opacity rule, but I did remove the color of `monaco.shell` and I have black text, as I wanted. 
Thanks! 
You may consider to fix it for other users too.
![svscode2](https://cloud.githubusercontent.com/assets/11356118/15023908/82055cd6-1200-11e6-80fb-bcc1d2d967b9.png)

**Comment by bpasero** [comment]
With the new options of themes to color the foreground text in sidebar (coming in May) I suggest to close this one and let users pick the best theme that matches expectations.

**Comment by biswajitpaul01** [comment]
You can also override your active theme using User Settings file. Have a look into this: https://code.visualstudio.com/docs/getstarted/theme-color-reference
